pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -648,6 +648,10 @@ class Build {
                         extra_app_options += " customJvmOpts=-Djava.net.preferIPv4Stack=true"
                     }
 
+                    if (platform.contains("windows") && jdkVersion >= 24) {
```

Comment:
This should be "extra_options" not "extra_app_options", there's a very subtle difference, "extra_app_options" also gets set on all other processes including the jck Agent process, this is currently only needed for AIX (see above).

I'll create a new PR to resolve

Reply:
Ok. Closing this PR.